packages/reassure-cli/src/types.ts

```diff
@@ -0,0 +1,3 @@
+export type DefaultOptions = {
+  logLevel?: string;
```

Comment:
I think we should rather go with `--verbose` and `--silent` options, as this is what most of the CLIs offer and will be familiar pattern for our users.

Reply:
done